frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -123,6 +147,7 @@ const deleteUserRoleAssignment = async (
 
 const deleteAppAdmin = async (admin: FamAppAdminGetResponse) => {
     resetNotification();
+    resetNewTag();
```

Comment:
so even though we can to keep `resetNotification()` and `resetNewTag()` as separate method, but inside this component, you can create a method to include two, and call this method everytime

const resetNotificationAndTag = () = > {
     resetNotification();
     resetNewTag();
}

Reply:
sure, I'll follow your suggestion